spec/dummy_nanoc/output/index.html

```diff
@@ -61,26 +61,26 @@ <h3>Live demo on these links :</h3>
 <b>Note : </b>There is some problem in nyaplot (in live link. It works fine locally). Some css is not working so some styling doesn't working properly. You can see it properly in local setup.
 </p>
 
-<a href="https://shekharrajak.github.io/daru-view/spec/dummy_nanoc/output/"> Index.html</a>
+<a href="https://sciruby.github.io/daru-view/spec/dummy_nanoc/output/"> Index.html</a>
 <br><br>
-<a href="https://shekharrajak.github.io/daru-view/spec/dummy_nanoc/output/nyaplot/">Nyaplot Examples </a><br><br>
-<a href="https://shekharrajak.github.io/daru-view/spec/dummy_nanoc/output/highcharts/">Highcharts Examples </a>
+<a href="https://sciruby.github.io/daru-view/spec/dummy_nanoc/output/nyaplot/">Nyaplot Examples </a><br><br>
+<a href="https://sciruby.github.io/daru-view/spec/dummy_nanoc/output/highcharts/">Highcharts Examples </a>
 <br><br>
-<a href="https://shekharrajak.github.io/daru-view/spec/dummy_nanoc/output/googlecharts/">Google Charts Examples </a>
+<a href="https://sciruby.github.io/daru-view/spec/dummy_nanoc/output/googlecharts/">Google Charts Examples </a>
```

Comment:
Okay! It look like you manually updated these lines, rather than command `bundle exec nanoc view`. I guess is also fine.